features/step_definitions/govspeak_steps.rb

```diff
@@ -6,6 +6,10 @@
   expect_to_see_a_govspeak_enabled_textarea_for_id("edition_details_rates_description")
 end
 
+Then("I see that the contact address description is Govspeak-enabled") do
```

Comment:
I wonder if it might be a bit more DRY to have a regex here? Something like:

```
Then(/^I see that the ([^"]*) is Govspeak-enabled$/) do |field|
  id = case field
         when "contact address description"
            "edition_details_addresses_description"
         ... etc
    end
    expect_to_see_a_govspeak_enabled_textarea_for_id(id)
end
```

Reply:
good idea, thks: it's nice to have all the IDs listed together